rewrite-json/src/main/java/org/openrewrite/json/JsonParser.java

```diff
@@ -75,7 +75,7 @@
 
     @Override
     public boolean accept(Path path) {
-        return path.toString().endsWith(".Json");
+        return path.toString().endsWith(".json");
```

Comment:
should we also be targeting json5 and jsonc ?
@jkschneider thoughts
```suggestion
        return path.toString().matches("(?i)\\.json(5|c)?$"); // REGEX: captures .json, .json5, .jsonc;
```